package.json

```diff
@@ -42,15 +42,16 @@
     "html2canvas": "^1.4.1",
     "jspdf": "^2.5.2",
     "lodash": "^4.17.21",
-    "next": "^14.2.6",
+    "next": "^14.2.13",
     "prettier": "^3.3.3",
     "react-blockies": "^1.4.1",
     "react-hook-form": "^7.47.0",
     "react-number-format": "^5.3.1",
     "styled-components": "^6.1.0",
     "tailwind-merge": "^2.5.2",
     "viem": "^2.20.1",
-    "wagmi": "^2.12.7"
+    "wagmi": "^2.12.7",
+    "yarn": "^1.22.22"
```

Comment:
Does `yarn` need to be added as a dependency???
- If `yes`, why is it needed???